charts/index.yaml

```diff
@@ -1,79 +1,104 @@
-      icon: https://timeplus.io/1_SYMBOL_B.png
-      keywords:
-        - timeplus
-        - realtime
-        - streaming
-        - analytics
-        - monitoring
-        - SQL
-      kubeVersion: ">= 1.25.0-0"
-      maintainers:
-        - email: eng@timeplus.com
-          name: Timeplus engineers
-      name: timeplus-enterprise
-      sources:
-        - https://github.com/timeplus-io/helm-charts
-      type: application
-      urls:
-        - https://install.timeplus.com/charts/timeplus-enterprise-v2.3.5.tgz
-      version: v2.3.5
-    - apiVersion: v2
-      appVersion: 2.2.5
-      created: "2024-07-11T23:04:56.836647-07:00"
-      description: A Helm chart for deploying a Timeplus enterprise stack.
-      digest: e9aa7d6e6f120529a6ec2f39ed0eb6277619c436faec70c2ac5181c8fd76f88b
-      home: https://www.timeplus.com/
-      icon: https://timeplus.io/1_SYMBOL_B.png
-      keywords:
-        - timeplus
-        - realtime
-        - streaming
-        - analytics
-        - monitoring
-        - SQL
-      kubeVersion: ">= 1.25.0-0"
-      maintainers:
-        - email: eng@timeplus.com
-          name: Timeplus engineers
-      name: timeplus-enterprise
-      sources:
-        - https://github.com/timeplus-io/helm-charts
-      type: application
-      urls:
-        - https://install.timeplus.com/charts/timeplus-enterprise-2.3.0.tgz
-      version: 2.3.0
-generated: "2024-07-11T23:04:56.835605-07:00"
+  - apiVersion: v2
+    appVersion: 2.3.20
+    created: "2024-08-01T14:39:27.048662-07:00"
+    description: Helm chart for deploying a cluster of Timeplus Enterprise.
+    digest: e84c65c0fecbd2bbcb36258c5645bf61fcf66a10355acd7d697a25d42bd75644
+    home: https://www.timeplus.com/
+    icon: https://timeplus.io/1_SYMBOL_B.png
+    keywords:
+    - timeplus
+    - realtime
+    - streaming
+    - analytics
+    - monitoring
+    - SQL
+    kubeVersion: '>= 1.25.0-0'
+    maintainers:
+    - email: eng@timeplus.com
+      name: Timeplus engineers
+    name: timeplus-enterprise
+    sources:
+    - https://github.com/timeplus-io/proton
+    type: application
+    urls:
+    - https://install.timeplus.com/charts/timeplus-enterprise-v2.4.15.tgz
+    version: v2.4.15
+  - apiVersion: v2
+    appVersion: 2.3.7
+    created: "2024-08-01T14:39:27.049627-07:00"
+    description: A Helm chart for deploying a cluster of Timeplus Enterprise.
+    digest: b3cc69f0cc7602eb40ed682fc950cc47122c3fd709d0246719ca85822b765bf1
+    home: https://www.timeplus.com/
+    icon: https://timeplus.io/1_SYMBOL_B.png
+    keywords:
+    - timeplus
+    - realtime
+    - streaming
+    - analytics
+    - monitoring
+    - SQL
+    kubeVersion: '>= 1.25.0-0'
+    maintainers:
+    - email: eng@timeplus.com
+      name: Timeplus engineers
+    name: timeplus-enterprise
+    sources:
+    - https://github.com/timeplus-io/proton
+    type: application
+    urls:
+    - https://install.timeplus.com/charts/timeplus-enterprise-v2.4.2.tgz
```

Comment:
Please remove the section for 2.4.2, since it's a intermediate release. Or you can merge the PR then I will delete this. (I deleted other internal releases yesterday and kept the 2.4.2 as the last 2.4.x build)

Reply:
I removed all older releases. Now 2.4.15 is the only chart

Reply:
Well, the doc page https://docs.timeplus.com/enterprise-v2.3 shows how to get the 2.3.0 and 2.3.5 helm charts since they are released before.

I don't have have very strong opinion but slightly prefer keeping the 2.3.0 and 2.3.5 as the older chart version. If you merge this as-is, then I will go ahead update the v2.3 doc not to mention the helm chart